packages/driver/src/cypress/error_messages.ts

```diff
@@ -1171,6 +1171,11 @@ export default {
     },
   },
 
+  network_logs: {
+    docsUrl: 'https://on.cypress.io/request-logs',
+    filter_must_be_function: ({ filterFn }) => `RequestLogs.filter should be set to a function, but a ${typeof filterFn} was passed.`,
```

Comment:
```suggestion
    filter_must_be_function: ({ filterFn }) => `RequestLogs.filter should be set to a function, but you tried to assign ${format(filterFn)}.`,
```

They tried to assign a value, not really pass something.

Reply:
Thanks, I missed this when moving from `.filter(newFilter)` to `.filter = newFilter`.